providers/google/newsfragments/quota_project.rst

```diff
@@ -0,0 +1,12 @@
+add-quota-project-support
```

Comment:
This file shouldn't exist, all the information about the PR and changes for the changelog will be prepared by release coordinator